doc/source/ray-core/fault-tolerance.rst

```diff
@@ -0,0 +1,63 @@
+Fault tolerance
+===============
+
+Ray is a distributed system, and that means failures can happen. Generally, failures can
+be classified into two classes: 1) failures internal to Ray, and 2) application-level failures. The first can be triggered by node failures,
+network failures, or just bugs in Ray. The latter can happen because of bugs in user-level code, or if external systems fail.
```

Comment:
If you going to use "first" then to be consistent use "The second" rather than "The latter"